core/src/main/resources/reference.conf

```diff
@@ -206,183 +69,219 @@ cassandra-journal {
   # Supported values: 3, 4
   protocol-version = ""
```

Comment:
I don't think we need this one any more created https://github.com/akka/akka-persistence-cassandra/issues/644